merkle/log_verifier_test.go

```diff
@@ -416,10 +416,60 @@ func TestVerifyConsistencyProof(t *testing.T) {
 	}
 }
 
+func TestPrefixHashFromInclusionProofGenerated(t *testing.T) {
+	sizes := make([]int64, 0, 17)
+	for s := 2; s <= 17; s++ {
```

Comment:
Might need some tests for invalid inputs.

Reply:
Done.